core/src/from_meta.rs

```diff
@@ -152,6 +157,20 @@ impl FromMeta for AtomicBool {
     }
 }
 
+impl FromMeta for char {
+    fn from_char(value: char) -> Result<Self> {
+        Ok(value)
+    }
+
+    fn from_string(s: &str) -> Result<Self> {
+        if s.chars().count() == 1 {
+            Ok(s.chars().next().unwrap())
+        } else {
+            Err(Error::unexpected_type("string"))
+        }
```

Comment:
Let's avoid unwrap here since we can
```suggestion
		let mut chars = s.chars();
		let char1 = chars.next();
		let char2 = chars.next();

        if let (Some(char), None) = (char1, char2) {
            Ok(char)
        } else {
            Err(Error::unexpected_type("string"))
        }
```